docs/docs/configuration/cache.md

```diff
@@ -9,24 +9,23 @@ The cache directory includes
 The cache option is common to all scanners.
 
 ## Clear Caches
-The `--clear-cache` option removes caches.
-
-**The scan is not performed.**
+`trivy clean` subcommand removes caches.
 
 ```
-$ trivy image --clear-cache
+$ trivy clean --scan-cache
```

Comment:
We say about removing cache**s**. I think it is better to use `-all` flag here.
```suggestion
$ trivy clean --all
```

Reply:
I also added `--all`.
https://github.com/aquasecurity/trivy/pull/6993/commits/361db194a15948ccd9a94b676d85b17431ca168b